apps/src/music/analytics/AnalyticsReporter.ts

```diff
@@ -267,6 +236,19 @@ export default class AnalyticsReporter {
   }
 
   log(message: string) {
-    console.log(`[AMPLITUDE ANALYTICS EVENT]: ${message}`);
+    console.log(`[MUSIC AMPLITUDE ANALYTICS EVENT]: ${message}`);
+  }
+
+  private formatUserId(userId: number) {
+    const userIdString = userId.toString() || 'none';
+    if (!userId) {
+      return userIdString;
+    }
```

Comment:
non-blocking, but this feels backwards to me. Would it make more sense as:

```
if (!userId) {
  return 'none'
}
const userIdString = userId.toString()
```

My thinking is that we're explicitly assuming the `userId` exists (and is truthy) on line 243, but then on 244 we're checking to see if it doesn't exist (or, is at least non-truthy) and then bombing out later.

Reply:
Oh yep good call - copied this from another class but didn't catch this, thanks